### PR TITLE
[tune] resources_per_trial from trial_resources

### PR DESCRIPTION
**Pull request by richardliaw**
Renaming variable due to user errors.

### CODE REVIEW

**Review by ericl — COMMENTED** [review]
Shall we make it logger.warn and raise later?

File: python/ray/tune/config_parser.py [hunk 1]
```diff
@@ -197,8 +197,9 @@ def create_trial_from_spec(spec, output_path, parser, **trial_kwargs):
         args = parser.parse_args(to_argv(spec))
     except SystemExit:
         raise TuneError("Error parsing args, see above message", spec)
-    if "trial_resources" in spec:
-        trial_kwargs["resources"] = json_to_resources(spec["trial_resources"])
+    if "resources_per_trial" in spec:
+        trial_kwargs["resources"] = json_to_resources(
```
Comment (ericl) [inline python/ray/tune/config_parser.py]: Does this actually parse the old value right? Can you add a test that using the old value is fine?

**Review by ericl — APPROVED** [review]

**Comment by visatish** [comment]
@richardliaw this looks good

File: python/ray/tune/config_parser.py (hunk 1 above)
Comment (richardliaw) [reply python/ray/tune/config_parser.py]: er the old value is just dropped; sure will add test

**Pull request merged**